app/controllers/api/v1/user_availabilities_controller.rb

```diff
@@ -16,10 +16,21 @@ def destroy
     render json: timeblock
   end
 
+  def multiple
+    ids = time_block_ids_params
+    ids.each do |id|
+      UserAvailability.create(user: current_user, time_block_id: id)
+    end
```

Comment:
should have been more specific 😅 
I think you can do `availability_params[:time_block_ids]` and that will already be a strong param

also if it's a single line better to write as `ids.each { |id| UserAvailability.create(user: current_user, time_block_id: id) }`

Reply:
> availability_params[:time_block_ids]

This didn't work.